test/code-cache/test-code-cache-generator.js

```diff
@@ -1,28 +1,40 @@
 'use strict';
```

Comment:
Seems like all `code-cache` tests were removed or moved besides this one. Is that intentional?

Reply:
I'll remove it since the build tool chains actually uses the tool, and compiles the output.